src/core/reducers/addons.js

```diff
@@ -77,7 +77,10 @@ export function fetchAddon({
   }
   return {
     type: FETCH_ADDON,
-    payload: { errorHandlerId: errorHandler.id, slug },
+    payload: {
+      errorHandlerId: errorHandler.id,
+      slug: typeof slug === 'string' ? slug.trim() : slug,
```

Comment:
We're unlikely to need to worry about `new String('whatever');`, but `typeof` on that would be 'object' rather than string.